h2o-core/src/main/java/hex/ModelMetricsBinomial.java

```diff
@@ -18,7 +18,7 @@
 public class ModelMetricsBinomial extends ModelMetricsSupervised {
   public final AUC2 _auc;
   public final double _logloss;
-  public final double _mean_per_class_error;
```

Comment:
Needed?

Reply:
Yes actually. In the two cases the final was removed, the value might be overridden in the `ModelMetricsBinomialGeneric` constructor.